proxies/dotnet/Models/ClientRequestUser.cs

```diff
@@ -1,57 +1,82 @@
-using System.Text.Json.Serialization;
 using Newtonsoft.Json;
-
-using DevCycle.SDK.Server.Cloud.Api;
-using DevCycle.SDK.Server.Local.Api;
-using DevCycle.SDK.Server.Common.Model;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
 using static DevCycle.SDK.Server.Common.Model.User;
 
+[DataContract]
 public class ClientRequestUser {
-    [JsonPropertyName("user_id")]
+    [JsonProperty("user_id")]
     public string? UserId { get; set; }
 
-    [JsonPropertyName("email")]
+    [JsonProperty("email")]
     public string? Email { get; set; }
 
-    [JsonPropertyName("country")]
+    [JsonProperty("country")]
     public string? Country { get; set; }
 
-    [JsonPropertyName("language")]
+    [JsonProperty("language")]
     public string? Language { get; set; }
 
-    [JsonPropertyName("name")]
+    [JsonProperty("name")]
     public string? Name { get; set; }
 
-    [JsonPropertyName("customData")]
+    [JsonProperty("customData")]
     public Dictionary<string, object>? CustomData { get; set; }
 
-    [JsonPropertyName("privateCustomData")]
+    [JsonProperty("privateCustomData")]
     public Dictionary<string, object>? PrivateCustomData { get; set; }
 
-    [JsonPropertyName("deviceModel")]
+    [JsonProperty("deviceModel")]
     public string? DeviceModel { get; set; }
 
-    [JsonPropertyName("platformVersion")]
+    [JsonProperty("platformVersion")]
     public string? PlatformVersion { get; set; }
 
-    [JsonPropertyName("platform")]
+    [JsonProperty("platorm")]
     public string? Platform { get; set; }
 
-    [JsonPropertyName("lastSeenDate")]
+    [JsonProperty("lastSeenDate")]
     public DateTime LastSeenDate { get; set; }
     
-    [JsonPropertyName("createdDate")]
+    [JsonProperty("createdDate")]
     public DateTime CreatedDate { get; set; }
 
-    [JsonPropertyName("appBuild")]
+    [JsonProperty("appBuild")]
     public double AppBuild { get; set; }
     
-    [JsonPropertyName("appVersion")]
+    [JsonProperty("appVersion")]
     public string? AppVersion { get; set; }
     
-    [JsonPropertyName("sdkType")]
+    [JsonProperty("sdkType")]
     public SdkTypeEnum? SdkType { get; set; }
 
-    [JsonPropertyName("sdkVersion")]
+    [JsonProperty("sdkVersion")]
     public string? SdkVersion { get; set; }
-}
\ No newline at end of file
+
+    public override string ToString()
+        {
+            var sb = new StringBuilder();
+            sb.Append("class UserData {\n");
```

Comment:
can we quickly change `UserData` in the string to `ClientRequestUser`?

Reply:
doone